doc/source/science_guide/sg_dynamics.rst

```diff
@@ -424,7 +469,7 @@ can be set in the namelist. The plastic potential can lead to more realistic fra
 
 By default, the namelist parameters are set to :math:`e_f=e_g=2` and :math:`k_t=0` which correspond to the standard VP rheology.
 
-There are four options in the code for solving the sea ice momentum equation with a VP formulation: the standard EVP approach, a 1d EVP solver, the revised EVP approach and an implicit Picard solver. The modifications to the yield curve and to the flow rule described above are available for these four different solution methods.  
+There are four options in the code for solving the sea ice momentum equation with a VP formulation: the standard EVP approach, a 1d EVP solver, the revised EVP approach and an implicit Picard solver. The choice of the capping method for the viscosities and the modifications to the yield curve and to the flow rule described above are available for these four different solution methods. Note that only the EVP and revised EVP methods are currently available if one chooses the C grid. 
```

Comment:
Farther up, I think you said that only EVP is available for the C-grid (not mentioning rEVP).

Reply:
Yes I think we need to introduce rEVP a little bit earlier. I will have to rework the intro of dynamics. Coming soon...